modules/ROOT/pages/_partials/gs-deploying-apicast-proc.adoc

```diff
@@ -0,0 +1,37 @@
+[id='gs-deploying-apicast-proc']
+
+ifdef::env-github[]
+:imagesdir: ../images/
+endif::[]
+
+// https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.5/html/installing_3scale/installing-apicast
+
+= Deploying an APIcast API gateway
+
+APIcast is an NGINX based API gateway used to integrate your internal and external API services with the 3scale Platform.
+
+There are two APIcast deployment options:
+
+*Embedded APIcast*: {PRODUCT_SHORT} hosts APIcast in the cluster. In this case, APIcast is already deployed for you, but is not suitable for a production environment.
+
+*Self-managed APIcast*: You can deploy APIcast wherever you want. The self-managed mode is the intended mode of operation for production environments.
```

Comment:
`intended mode of operation` only applies to 3scale SaaS, so it's not relevant to us IIRC.

Reply:
removed all mentions of 'mode'